tests/show-config-fixtures/empty.txt

```diff
@@ -59,24 +59,6 @@ filter-host = []
 filter-label = []
 filter-paths = []
 filter-tags = []
-keep-tags = []
-keep-ids = []
-keep-last = 0
-keep-hourly = 0
-keep-daily = 0
-keep-weekly = 0
-keep-monthly = 0
-keep-quarter-yearly = 0
-keep-half-yearly = 0
-keep-yearly = 0
-keep-within = "0s"
-keep-within-hourly = "0s"
-keep-within-daily = "0s"
-keep-within-weekly = "0s"
-keep-within-monthly = "0s"
-keep-within-quarter-yearly = "0s"
-keep-within-half-yearly = "0s"
-keep-within-yearly = "0s"
```

Comment:
Could you please give some context as for why you needed to remove these? I am wondering ^^

Reply:
I changed the `KeepOptions` in rustic_core in https://github.com/rustic-rs/rustic_core/pull/238. There also the Serialization was changed to skip them if the Options are `None`. The serialization is actually only relevant for the `show-config` command in rustic (and for the unit tests added in that PR).
For `show-config` we do have integration tests here in rustic, so I needed to change the expected output - after updating to the rustic_core version which includes that PR.

Reply:
Thanks I understand better!